dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi-concurrent3.0.feature

```diff
@@ -1,10 +1,9 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.cdi-concurrent3.0
 visibility=private
-#TODO remove temporary API package for simulating possible additions to Jakarta Concurrency
-IBM-API-Package: prototype.enterprise.concurrent; type="ibm-api"
+#TODO remove osgi.identity=io.openliberty.cdi-3.0
 IBM-Provision-Capability: \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-4.0))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.cdi-4.0)(osgi.identity=io.openliberty.cdi-3.0))", \
```

Comment:
It looks like you are missing an ending parentheses: 
```
(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.cdi-4.0)(osgi.identity=io.openliberty.cdi-3.0)))
```

Reply:
Good catch, I added a commit to correct it